api/tests/integration/infrastructure/utils/bookshelf-to-domain-converter_test.js

```diff
@@ -61,8 +61,9 @@ describe('Integration | Infrastructure | Utils | Bookshelf to domain converter',
     it('should support has-one relationships', async () => {
       // given
       const userId = databaseBuilder.factory.buildUser().id;
-      const assessmentId = databaseBuilder.factory.buildAssessment({ userId }).id;
-      databaseBuilder.factory.buildCampaignParticipation({ assessmentId, userId });
+      const campaignParticipation = databaseBuilder.factory.buildCampaignParticipation({ userId });
```

Comment:
Le titre du test est `should support has-one relationships`… mais cette relation n'est plus `hasOne` mais `belongsTo` 😮